W-16026636 Making fixes (#341)
mulesoft/docs-access-management: modules/ROOT/pages/verify-saml-info-task.adoc

= Verifying SAML Information
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> During verification of SAML information, if you determine that information needs to be changed, sign into the MuleSoft Support Portal and submit a request.

Reviewer: ```suggestion
During verification of SAML information, if you determine that information must be changed, sign in to the MuleSoft Support Portal and submit a request.
```

Reviewer: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22signin,%22%22signout%22
